src/puppetlabs/puppetdb/metrics/core.clj

```diff
@@ -7,7 +7,8 @@
     {:registry registry
      :reporter (reporter registry {:domain domain})}))
 
-(def metrics-registries {:mq (new-metrics "puppetlabs.puppetdb.mq")
+(def metrics-registries {:clean (new-metrics "puppetlabs.puppetdb.clean")
```

Comment:
That'd be fine with me - didn't know how we wanted to handle the (naming) scopes.